test/cppsuite/test_harness/workload/thread_context.h

```diff
@@ -135,10 +135,8 @@ class thread_context {
      * Return true if the operation was successful, a return value of false implies the transaction
      * needs to be rolled back.
      */
-    bool insert(
-      scoped_cursor &cursor, uint64_t collection_id, uint64_t key_id, wt_timestamp_t ts = 0);
-    bool insert(
-      scoped_cursor &cursor, uint64_t collection_id, const std::string &key, wt_timestamp_t ts = 0);
+    bool insert(scoped_cursor &cursor, uint64_t collection_id, uint64_t key_id);
+    bool insert(scoped_cursor &cursor, uint64_t collection_id, const std::string &key);
```

Comment:
Why did you remove timestamps from inserts?
Also if timestamps are gone, the comment for the methods needs to be updated.

Reply:
@AlexeyAnisimovAU, it is not used anywhere, some code cleanup.
Thanks for spotting the incorrect comment, updated in 4ff102d485eb4b260573116f0ba5602b7cc15da6.